src/Jackett.Common/Definitions/exitorrent-org.yml

```
@@ -0,0 +1,156 @@
+---
+id: exitorrent-org
+name: exitorrent.org
+description: "exitorrent.org is a POLISH Private Torrent Tracker"
+language: pl-PL
+type: private
+encoding: UTF-8
+links:
+  - http://exitorrent.org/
```

The site supports secure connections so lets use it
```suggestion
  - https://exitorrent.org/
```